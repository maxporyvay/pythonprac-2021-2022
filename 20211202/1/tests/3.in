W = dump('W', (), {'methodd':lambda self, x, **kw: x+sum(v for k,v in kw.items())})
a = W()
print(a.methodd(5, e=7, u=78))