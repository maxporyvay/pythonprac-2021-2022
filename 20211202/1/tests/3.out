methodd: (5,), {'e': 7, 'u': 78}
90
